Gemfile

```diff
@@ -3,8 +3,8 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in email_repair.gemspec
 gemspec
 
+gem 'coveralls'
 gem 'rake'
 gem 'rspec'
-gem 'rubocop', '~> 0.27.0'
+gem 'rubocop'
```

Comment:
We probably want to have some sort of version constraint here. Really all of our dev dependencies should have version constraints.